command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@wking said: `Agreed. Can you point out where I'm suggesting changing the semantics
of an older spec?`

Agreed, good. As to semantics change, I didn't know which direction you were heading,  changing the runtime-spec to have a testing shim type requirement or using this runtime client spec to show the requirement for the testing shim. If the former, it's cool, if the later that would be the semantic change I was pointing to.